controllers/goharbor/chartmuseum/deployments.go

```diff
@@ -124,6 +124,40 @@ func (r *Reconciler) GetDeployment(ctx context.Context, chartMuseum *goharborv1.
 		})
 	}
 
+	// refer https://github.com/helm/chartmuseum/blob/main/README.md and https://github.com/helm/chartmuseum/blob/main/pkg/config/vars.go
+	if chartMuseum.Spec.Chart.Storage.AliOSS != nil { //nolint:dupl
+		envs = append(envs, corev1.EnvVar{
+			Name:  "STORAGE",
+			Value: "alibaba",
+		}, corev1.EnvVar{
+			Name:  "STORAGE_ALIBABA_BUCKET",
+			Value: chartMuseum.Spec.Chart.Storage.AliOSS.Bucket,
+		}, corev1.EnvVar{
+			Name:  "STORAGE_ALIBABA_ENDPOINT",
+			Value: chartMuseum.Spec.Chart.Storage.AliOSS.Endpoint,
+		}, corev1.EnvVar{
+			Name:  "ALIBABA_CLOUD_ACCESS_KEY_ID",
+			Value: chartMuseum.Spec.Chart.Storage.AliOSS.AccessKeyID,
+		}, corev1.EnvVar{
+			Name:  "STORAGE_ALIBABA_PREFIX",
+			Value: getChartFolder(chartMuseum.Spec.Chart.Storage.AliOSS.PathPrefix),
+		})
```

Comment:
Why is `STORAGE_ALIBABA_SSE` not included?
https://github.com/helm/chartmuseum/blob/main/pkg/config/vars.go#L463

Seems like it would be a nice setting to expose. Or are there reasons why it's missing here?

Reply:
because harbor-helm doesn't implement this.
https://github.com/goharbor/harbor-helm/blob/4f622fb27b6dde783c81aa99bae3d59cec6b400b/templates/chartmuseum/chartmuseum-cm.yaml#L107

Reply:
and registry storage setting doesn't have this setting too.
https://docs.docker.com/registry/storage-drivers/oss/
So I think this feature can be considered to add in future release if needed.

Reply:
I just add ServerSideEncryption in ChartMuseumChartStorageDriverOssSpec